### PR TITLE
Feature/prod env

### PR DESCRIPTION
**Pull request by umayrsNHS**
## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: .github/workflows/prod.yaml [hunk 1]
```diff
@@ -0,0 +1,217 @@
+name: "Prod"
+
+# The total recommended execution time for the "CI/CD Pull Request" workflow is around 20 minutes.
+
+on:
+  workflow_dispatch:
+    inputs:
+      invitations_branch:
+        description: "This is the branch/tag of Galleri-Invitations be deployed to the Performance environment"
```
Comment (abdulqadir-qasim-NHS) [inline .github/workflows/prod.yaml:9]: does this need to say "Production environment"?

File: .github/workflows/prod.yaml [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+        required: true
+      frontend_branch:
+        description: "This is the branch/tag of the Galleri frontend to be deployed to the Performance environment"
```
Comment (abdulqadir-qasim-NHS) [inline .github/workflows/prod.yaml:12]: same with this one

File: .github/workflows/prod.yaml [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+      nodejs_version: "${{ needs.metadata.outputs.nodejs_version }}"
+      python_version: "${{ needs.metadata.outputs.python_version }}"
+      terraform_version: "${{ needs.metadata.outputs.terraform_version }}"
+      version: "${{ needs.metadata.outputs.version }}"
+      environment: ${{ needs.metadata.outputs.environment }}
+      invitations_branch: "${{ github.event.inputs.invitations_branch }}"
+      frontend_branch: "${{ github.event.inputs.frontend_branch }}"
+      tf_action: "${{ github.event.inputs.tf_action }}"
+    secrets:
+      AWS_ACCOUNT_ID: ${{ secrets.PRE_PROD_AWS_ACCOUNT_ID }}
```
Comment (abdulqadir-qasim-NHS) [inline .github/workflows/prod.yaml:161]: Would this need to be the PROD aws account id?

File: .github/workflows/prod.yaml [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+      build_datetime: "${{ needs.metadata.outputs.build_datetime }}"
+      build_timestamp: "${{ needs.metadata.outputs.build_timestamp }}"
+      build_epoch: "${{ needs.metadata.outputs.build_epoch }}"
+      nodejs_version: "${{ needs.metadata.outputs.nodejs_version }}"
+      python_version: "${{ needs.metadata.outputs.python_version }}"
+      terraform_version: "${{ needs.metadata.outputs.terraform_version }}"
+      version: "${{ needs.metadata.outputs.version }}"
+      environment: ${{ needs.metadata.outputs.environment }}
+    secrets:
+      AWS_ACCOUNT_ID: ${{ secrets.PRE_PROD_AWS_ACCOUNT_ID }}
```
Comment (abdulqadir-qasim-NHS) [inline .github/workflows/prod.yaml:194]: Also this one

File: .github/workflows/prod.yaml [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
```
Comment (abdulqadir-qasim-NHS) [inline .github/workflows/prod.yaml:210]: Change to PROD here too?

File: .github/workflows/prod.yaml [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+        description: "Determines if the seeding scripts are run to populate the database"
+        type: choice
+        required: true
+        default: "yes"
+        options:
+          - "yes"
+          - "no"
+      tf_action:
+        type: choice
+        description: Terraform action to be carried out against UAT environment
```
Comment (abdulqadir-qasim-NHS) [inline .github/workflows/prod.yaml:24]: Would this need to be PROD environment?

**Review by abdulqadir-qasim-NHS — APPROVED** [review]

**Review by Amar-NHS — APPROVED** [review]

**Pull request merged**